src/qml/editorwidgets/ValueRelation.qml

```diff
@@ -1,10 +1,11 @@
-import QtQuick 2.11
-import QtQuick.Controls 2.4
+import QtQuick 2.12
+import QtQuick.Controls 2.12
 
 
 import QtGraphicalEffects 1.0
 import QtQuick.Layouts 1.0
 import ".."
+import "."
```

Comment:
Is this required?

For the record, I'd like to get rid of these `.` and `..` imports in favor of imports of Theme if feasible

Reply:
fixed